app/assess/routes.py

```diff
@@ -124,7 +124,7 @@ def display_sub_criteria(
 
         comment_form = CommentsForm()
 
-        if comment_form.validate_on_submit():
+        if request.method == "POST" and comment_form.validate_on_submit():
```

Comment:
From the docs [validate_on_submit()](https://flask-wtf.readthedocs.io/en/0.15.x/quickstart/#validating-forms) also checks if the request.method is post so this shouldn't be needed I think?